front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -71,15 +75,25 @@ const StcdmResults = ({
     useState(SPEED_SPACE_CHART_HEIGHT);
 
   const selectedSimulation = simulationsList[selectedSimulationIndex];
-  const spaceTimeData = useProjectedTrainsForStdcm(selectedSimulation.outputs?.results);
+  const hasConflicts = (selectedSimulation?.conflicts?.length ?? 0) > 0;
```

Comment:
I think we should put a more explicit variable name here

```suggestion
  const  hasConflictOnSelectedSimulation = (selectedSimulation?.conflicts?.length ?? 0) > 0;
```

Reply:
Variable used differently can you check now with the usage of the type guard ?

Reply:
Done